docs/cpk.md

```diff
@@ -0,0 +1,139 @@
+# CPK methods
+
+## create
+```
+static async create(opts?: CPKConfig): Promise<CPK>
+```
+Static method that creates and initializes an instance of the CPK with the selected configuration parameters.
+
+## constructor
+```
+constructor(opts?: CPKConfig)
+```
+Creates a non-initialized instance of the CPK with the selected configuration parameters.
+
+## isProxyDeployed
+```
+async isProxyDeployed(): Promise<boolean>
+```
+Returns true or false if the Safe Proxy used by the CPK is already deployed (at least one transaction was made) or if it is not (no transactions made yet).
+
+## isSafeApp
+```
+isSafeApp(): boolean
+```
+Returns true or false depending if the CPK is running as a Safe App (integrated in the Safe Web UI) or as a standalone app.
+
+## getOwnerAccount
+```
+async getOwnerAccount(): Promise<Address | undefined>
+```
+Returns the address of the account connected to the CPK. If the CPK is connected to a Safe or running as a Safe App the Safe address will be returned.
+
+## ethLibAdapter
+```
+get ethLibAdapter(): EthLibAdapter | undefined
+```
+Returns the EthLibAdapter in use.
+
+## networks
+```
+get networks(): NetworksConfig
+```
+Returns the network configuration in use.
```

Comment:
"network**s** configuration"? Maybe call it a listing of the addresses of contracts which drive the CPK per network by network ID?